docs/web-apps/docker.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Docker
+current_menu: web-docker
+previous:
+    link: docs/web-apps/local-development.html
+    title: Local development for web apps
+---
+
+AWS Lambda support running a Docker Container as a handler
+for your function. This alternative allows for lambda functions
+with up to 10GB to be deployed, as opposed to 250MB for non-docker.
+Another reason to choose this approach is if you need more than 5
+PHP extensions. Installing/Enabling multiple extensions on Docker
+is easier and will avoid hitting the limit of 5 layers per function.
+
+## Docker Image
+
+Bref helps you deploy Docker to AWS Lambda by offering
+out-of-the-box base images that are package for the Lambda environment.
+Here is an example of a Docker image
+
+```Dockerfile
+FROM composer:2 as installation
+
+WORKDIR /app
+
+RUN composer require bref/bref
+
+FROM bref/php-80-fpm
+
+COPY --from=bref/extra-redis-php-80:0.10 /opt/bref-extra /opt/bref-extra
+
+COPY --from=bref/extra-gmp-php-80:0.10 /opt/bref-extra /opt/bref-extra
+COPY --from=bref/extra-gmp-php-80:0.10 /opt/bref/ /opt/bref
+
+COPY --from=installation /app/ /var/task/
+
+RUN echo "<?php echo 'Hello from Docker Bref!';" >> /var/task/public/index.php
+
+CMD ["public/index.php"]
```

Comment:
Really cool!

I would suggest splitting the example like this:

- first, the docs should start with the simplest example possible -> that example helps reader determine whether they get it or not. I think the 1st example could fit in maybe 3 lines
- then, extra examples that address different problems: 1 example on "how can I run composer install in docker?", 1 example on "how can I install extra extensions"

I would also document how to deploy an existing project, not one that is created (no `composer require bref` + `echo`). Because I expect that at this point, people reading that documentation will want to upload `.` (their existing project) to Lambda.

Reply:
Also just to make it more explicit:

```suggestion
# Configure the handler file (the entrypoint that receives all HTTP requests)
CMD ["public/index.php"]
```